src/core/query/query.js

```diff
@@ -283,7 +273,7 @@ export default class Query {
    * @returns {Query} The query.
    */
   greaterThan(field, value) {
-    if (isNumber(value) === false && isString(value) === false) {
+    if (isNumber(value) === false && typeof value !== 'string') {
```

Comment:
Same 3 lines in 4 places in this file - perhaps this should be a utility function or at least a validation function for this class.

Reply:
Not sure I follow the comment "same 3 lines in 4 places in this file". There is some repeating that we can look at moving to a utility function in a future PR.